CRVResponse/src/MakeCrvPhotons.cc

```diff
@@ -469,40 +475,46 @@ bool MakeCrvPhotons::IsInsideFiber(const CLHEP::Hep3Vector &p, const CLHEP::Hep3
   return true;
 }
 
-double MakeCrvPhotons::GetRandomTime(const LookupBin *theBin, bool &overflow)
+double MakeCrvPhotons::GetRandomTime(const LookupBin *theBin)
 {
-  //the lookup tables encodes probabilities as probability*probabilityScale(10000), 
-  //so that the probabilities can be stored as integers.
-  //therefore, the probability of 1 is stored as 10000.
-  double rand=_randFlat.fire()*LookupBin::probabilityScale;
-  double sumProb=0;
+  //The lookup tables encodes probabilities as probability*mu2eCrv::LookupBin::probabilityScale(255), 
```

Comment:
Where is the scale value 255 (256) defined in code?

Reply:
In CRVResponse/inc/MakeCrvPhotons.hh line 72